aequilibrae/parameters.py

```diff
@@ -54,12 +53,38 @@ def __init__(self, project=None):
             self.parameters = deepcopy(self._default)
 
     def write_back(self):
-        """Writes the parameters back to file"""
+        """Writes the parameters back to file
```

Comment:
If we make sure that this is code that can be run, then we can actually test the docstrings.

What we can do is, in the beginning of the tests (before pytest), we create an example in the tempfolder (gettempdir()/"sfalls_just_to_open") and we can use that example everywhere.